packages/playwright-core/src/server/chromium/crPage.ts

```diff
@@ -543,7 +543,7 @@ class FrameSession {
       const options = this._crPage._browserContext._options;
       if (options.bypassCSP)
         promises.push(this._client.send('Page.setBypassCSP', { enabled: true }));
-      if (options.ignoreHTTPSErrors)
+      if (options.ignoreHTTPSErrors || options.ignoreHTTPSErrorsOverride)
```

Comment:
This line looks strange, because if `ignoreHTTPSErrorsOverride === false`, I'd assume we should not ignore the errors. I'd recommend to rename it to `internalIgnoreHTTPSErrors` to avoid this confusion.

Reply:
done.